js/ui/shared/accessibility.js

```diff
@@ -2,18 +2,48 @@ import $ from "../../core/renderer";
 import eventsEngine from "../../events/core/events_engine";
 import eventUtils from "../../events/utils";
 
-const FOCUS_STATE_CLASS = "dx-state-focused";
+const FOCUS_STATE_CLASS = "dx-state-focused",
+    FOCUS_DISABLED_CLASS = "dx-cell-focus-disabled",
+    DATA_ROW_SELECTOR = ".dx-datagrid-rowsview .dx-datagrid-content .dx-data-row",
+    viewItemSelectorMap = {
+        headerColumns: [".dx-datagrid-headers .dx-header-row > td.dx-datagrid-action"],
+        filterRow: [".dx-datagrid-headers .dx-datagrid-filter-row .dx-editor-cell input"],
+        rowsView: [`${DATA_ROW_SELECTOR} > td[tabindex]`, `${DATA_ROW_SELECTOR} > td`],
+        footer: [".dx-datagrid-total-footer .dx-datagrid-summary-item"],
+        filterPanel: [".dx-datagrid-filter-panel .dx-icon-filter"],
+        pager: [".dx-datagrid-pager [tabindex]", ".dx-datagrid-pager .dx-selection"]
```

Comment:
Is ".dx-datagrid-pager .dx-selection" selector necessary?

Reply:
Fixed